lib/nerves_runtime/init.ex

```diff
@@ -98,11 +98,19 @@ defmodule Nerves.Runtime.Init do
 
   defp unmount_if_error(s), do: s
 
-  defp format_if_unmounted(%{mounted: :unmounted} = s) do
+  defp format_if_unmounted(%{mounted: :unmounted, fstype: "f2fs"} = s) do
+    "Formatting application partition."
```

Comment:
I looked through the f2fs-tools source and it calls `uuid_generate(sb->uuid);`. This means that it's susceptible to the urandom pool initialization hang. Since you deleted that warning, did you find something different?

Reply:
Good catch, I've updated to keep the messages the warning the same.